File: Lab 3/Task3.go
package main

import "fmt"

func main() {
  var a, b int

  fmt.Println("Input 2 integers:")
  fmt.Scanf("%d %d", &a, &b)
  // NOTE: There is no ternary testing (?:) operation in Go.
  // -- https://golang.org/doc/faq#Does_Go_have_a_ternary_form
  fmt.Printf("The bigger one is %d\n", bigger(a, b))
}

func bigger(a, b int) int {
  if a >= b {
    return a
  } else {
    return b
  }
}

File: Lab 3/Task3.c
#include <stdio.h>

int main (void)
{
  int a = 0, b = 0;

  puts ("Input 2 integers:");
  scanf ("%d %d", &a, &b);
  printf ("The bigger one is %d\n", a >= b ? a : b);

  return 0;
}
